### ISSUE TITLE
eThe Bullets and numberingdialog in MS Word 2013 is anounced in a vague manner

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3417]
**Reported by sumandogra on 2013-08-07 15:07**
The bullets and numbering dialog in MS Word 2013 is very confusing now.  NVDA reads arbitrarily and one cannot find out which number or bullet  type one is on.
Steps to reproduce:
1)  Open a document that has a list to be numbered.
2)  Press ALT, O, N.
3)  Bullets and numbering dialog opens.
4)  Try to navigate in the dialog with TAB and ARROWS.
5)  On reaching the bullets or number types, NVDA reads arbitrarily and it becomes very difficult to find out which type of bullet or number one is on.

**Comment by bhavyashah** [comment]
@jcsteh @michaelDCurran Could you please respond to https://github.com/nvaccess/nvda/issues/3417#issuecomment-155308699 by sharing how this Word accessibility deficiency can be reported to Microsoft?

**Comment by michaelDCurran** [comment]
This is still broken in Office 365 and affects Narrator as well. I shall 
report this to Microsoft myself along with my other MS Word work.

**Comment by Adriani90** [comment]
@michaelDCurran any news from Microsoft regarding this issue?

**Comment by Adriani90** [comment]
I cannot really reproduce it in Office 2016 / 365.
@feerrenrut, @Qchristensen are you able to reproduce it in Office 2013 or older?

**Comment by Adriani90** [comment]
I don't think this is related, but When I access the dialog "define new numbering format", I get following error in my log when navigating with tab and shift+tab:

```
IO - inputCore.InputManager.executeGesture (20:42:38.082):
Input: kb(laptop):tab
DEBUGWARNING - NVDAObjects.window.winword.WordDocument._get_WinwordWindowObject (20:42:38.150):
Could not get MS Word object model from window 1248014 with class _WwN
Traceback (most recent call last):
  File "NVDAObjects\window\winword.pyc", line 1193, in _get_WinwordWindowObject
  File "oleacc.pyc", line 237, in AccessibleObjectFromWindow
  File "_ctypes/callproc.c", line 946, in GetResult
WindowsError: [Error -2147467259] Unbekannter Fehler
ERROR - scriptHandler.executeScript (20:42:38.161):
error executing script: <bound method Dynamic_WordDocumentEditableTextWithoutAutoSelectDetectionUIAWindowNVDAObject.script_tab of <NVDAObjects.Dynamic_WordDocumentEditableTextWithoutAutoSelectDetectionUIAWindowNVDAObject object at 0x053DD750>> with gesture u'Tab'
Traceback (most recent call last):
  File "scriptHandler.pyc", line 192, in executeScript
  File "NVDAObjects\window\winword.pyc", line 1371, in script_tab
  File "documentBase.pyc", line 24, in makeTextInfo
  File "NVDAObjects\UIA\__init__.pyc", line 285, in __init__
NotImplementedError: UIAutomationTextRangeArray is empty
```

**Comment by Adriani90** [comment]
@feerrenrut, @Qchristensen I wonder if this is reproducible in Office 2013 with NVDA 2019.2? Could you test this? In Office 2016 and 365 it is not.

**Comment by bdorer** [comment]
I could test this but I don't know which ribbons to use. The noted 
Shortcut isn't working. Could someone describe the way using ribbon names?

Thanks

Am 15.08.2019 um 22:57 schrieb Adriani90:

**Comment by Qchristensen** [comment]
I only currently have Office 365, but using that:
- Alt+o, n uses an old Office keystroke to open a bullets and numbering dialog.  Once it opens, it reads the item currently in use (for standard filled circle bullets made by typing * then space, this is now a "custom" bullet style in this dialog), but I can't get it to read the other parts of the dialog.

- Using the regular alt+h, u for bullets (or alt+h, n for numbering) the ribbon opens and reads the style currently being used.  @feerrenrut have you still got Office 2013 to test this?

**Comment by Adriani90** [comment]
The bullet and numbering dialog in Office 2013 has been splitted into two dialogs each for numbering and bullet in later versions of MS Office.
End of life for MS Office 2013 is officially on April 11, 2023. Thus I am closing this issue as abandoned. If you have the same issue in later version of MS Office (beginning with MS office 2016), please open a new issue covering those versions.
